### PR TITLE
Fix broken anchors on links

### PR DESCRIPTION
**Pull request by decrecementofeliz**

### CODE REVIEW

**Review by avivmu — APPROVED** [review]

File: files/en-us/learn/accessibility/wai-aria_basics/index.html [hunk 1]
```diff
@@ -263,7 +263,7 @@ <h3 id="Enhancing_keyboard_accessibility">Enhancing keyboard accessibility</h3>
  <li><code>tabindex="-1"</code> — this allows not normally tabbable elements to receive focus programmatically, e.g. via JavaScript, or as the target of links. </li>
 </ul>
 
-<p>We discussed this in more detail and showed a typical implementation back in our HTML accessibility article — see <a href="/en-US/docs/Learn/Accessibility/HTML#building_keyboard_accessibility_back_in">Building keyboard accessibility back in</a>.</p>
+<p>We discussed this in more detail and showed a typical implementation back in our HTML accessibility article — see <a href="/en-US/docs/Learn/Accessibility/HTML#Building_keyboard_accessibility_back_in">Building keyboard accessibility back in</a>.</p>
```
Comment (nschonni) [inline files/en-us/learn/accessibility/wai-aria_basics/index.html:266]: Flaw fixing will lowercase this again since they're case insensitive

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Yeah, because of how the system works (what @nschonni said), it'd be a better fix to update the ID of the "Building keyboard accessibility back in" heading on https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML instead.

Sorry for the confusion here!

**Comment by decrecementofeliz** [comment]
Thank you both, I've created a new PR that changes the HTML file. I don't know if I did this right (quite new with PR and git stuff), maybe I should I have edited my patch-1 branch, remove the first commit and the new changes there, and then update the PR?? (if that's even possible)?
Thank you

**Comment by chrisdavidmills** [comment]
> Thank you both, I've created a new PR, #2441 , that changes the link in the HTML file. I don't know if I did this right (quite new with PR and git stuff), maybe I should I have edited my patch-1 branch, remove the first commit and the new changes there, and then update the PR?? (if that's even possible)?
> Thank you

No worries at all here. Ideally, the best practice is to make further changes in the same branch, and then push those like you did with the original change — the changes are then added to the existing pull request automatically. You don't really need to remove the first commit — you can just keep adding changes on top until you are happy with it.

But for a simple change like this, making a new PR is an OK way to do it. GitHub is a hard tool to learn, so don't worry if you don't get everything perfect the first time.

So I guess this means we can close this one now. Thanks again!

**Comment by Ryuno-Ki** [comment]
> I don't know if I did this right (quite new with PR and git stuff), maybe I should I have edited my patch-1 branch, remove the first commit and the new changes there, and then update the PR?? (if that's even possible)?

In case you're still interested in how this could be done:
There's a feature called „interactive rebase”, where you pick the parent commit (I normally go with the commit previous to my changes) and then run it (on a command line with `git rebase -i <parent_commit_hash>` - but IDEs offer a visual way, too).

A window shows up which offers you different options. You can delete a line to remove that commit from history (kind of. If you recall the hash, you can still find it). You can also change the order, rename the commit message and other things.

Once you're done, save the changes and close the window.

Now the downside: You'll have to do a „force push”. This will overwrite the git history of your branch here on GitHub. So it can cause a mess for everybody who happened to have checked out your branch.

On the other hand, it's by convention, that those patch branches are living for a short time before they eventually get deleted, so the risk is small.

If you can share what editor / IDE you're using (VS Code, WebStorm, Vim, Sublime, Atom, IntelliJ etc) I can look up specific docs on this matter for you :)

**Comment by decrecementofeliz** [comment]
Hi thank you very much for your explanations. As IDE I've been using Github Desktop in some simple projects, now that I read @Ryuno-Ki 's comment, I realize I may have already "rebased" something in some of them, without knowing what rebasing was. I know now, so thank you!

In this particular case I sent the commits directly through the GitHub website and IDE.
Thanks again

**Pull request closed without merging**